ores/applications/celery.py

```diff
@@ -0,0 +1,58 @@
+"""
+Runs a celery worker.
+
+Usage:
+    celery [--config-dir=<path>]... [--logging-config=<path>]
+
+Options:
+    -h --help                Prints this documentation
+    --config-dir=<path>      The path to a directory containing configuration
+                             [default: config/]
+    --logging-config=<path>  The path to a logging configuration file
+"""
+import glob
+import logging
+import logging.config
+import os.path
+
+import docopt
+import yamlconf
+
+from ..score_processors import Celery
+
+DEFAULT_DIRS = ["config/", "/etc/ores/"]
+
+
+def main(argv=None):
+    args = docopt.docopt(__doc__, argv=argv)
+
+    run(config_dirs=args['--config-dir'],
+        logging_config=args['--logging-config'])
+
+
+def run(*args, **kwargs):
+    application = build(*args, **kwargs)
```

Comment:
In the utilities in the project, we split param parsing and cleanup from actually running a script.  This is the separation of responsibilities between main() and run().  Generally, it's best to pass *args, **kwargs rather than explicit arguments.  If run() needs an argument that build() can't take, that can still be handled semi-explicitly.  

E.g. 
```
def run(*args, runs_new_arg=None, **kwargs):
    ...
```